aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -522,6 +548,14 @@ define(["dojo/_base/declare",
          this.imgAltText = "";
          this.imgTitle = "";
 
+         // See AKU-941 - support for smart folders
+         if (!this.folderImageAspectMappings)
```

Comment:
No, I don't think so at this stage - because then we'd have to support clever ways of removing existing mappings, etc.